R/xl_open.R

```diff
@@ -1,42 +1,43 @@
-#' Open a Microsoft Excel file (xls/xlsx) or an openxlsx2 wbWorkbook
+#' Open an xlsx file or a `wbWorkbook` object
 #'
 #' @description
 #' This function tries to open a Microsoft Excel (xls/xlsx) file or,
-#' an openxlsx2 wbWorkbook with the proper application, in a portable manner.
+#' an [openxlsx2::wbWorkbook] with the proper application, in a portable manner.
 #'
 #' On Windows it uses `base::shell.exec()` (Windows only function) to
 #' determine the appropriate program.
 #'
-#' In Mac (c) it uses system default handlers, given the file type.
+#' On macOS, (c) it uses system default handlers, given the file type.
```

Comment:
Apple tends to change the name for the OS every now and then. Mac is more general I assume. Previously it was (Mac) OS X.